### PR TITLE
Add Dockerfile to build dockerfile-add-scaned as docker image

### PR DESCRIPTION
**Pull request by mikem-zed**
We need this tool to run in eve-kernel repo to generate spdx.json file
Someone (@eriknordmark ?)  with permissions should build the image and push it to lf-edge docker hub

### CODE REVIEW

**Review by deitch — APPROVED** [review]
Looks fine to me. We need to have some way to make deploying te image part of a pipeline as well.

File: tools/dockerfile-add-scanner/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Use an official Golang runtime as a parent image
+FROM golang:latest as builder
```
Comment (eriknordmark) [inline tools/dockerfile-add-scanner/Dockerfile:2]: Can we pin this to 1.20.1 which is what we use elsewhere?
And ideally use some ENV variable called GOVER to make it easier to find when we want to update?

**Review by eriknordmark — COMMENTED** [review]
Looks good to me except the use of latest golang.

File: tools/dockerfile-add-scanner/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+WORKDIR /go/src/app
+
+# Copy the current directory contents into the container at /go/src/app
+COPY . /go/src/app
+
+# Install any needed dependencies
+RUN go get -d -v ./...
+
+# Compile the application
+#RUN go build -o app
```
Comment (rene) [inline tools/dockerfile-add-scanner/Dockerfile:14]: It can be removed if is not needed...

File: tools/dockerfile-add-scanner/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+RUN GOOS=linux CGO_ENABLED=0 go build -o /bin/dockerfile-add-scanner
+
+# Deploy the application binary into a lean image
+FROM scratch
+
+WORKDIR /bin
+
+COPY --from=builder /bin/dockerfile-add-scanner /bin/dockerfile-add-scanner
+
+#USER nonroot:nonroot
```
Comment (rene) [inline tools/dockerfile-add-scanner/Dockerfile:24]: Same here

File: tools/dockerfile-add-scanner/Dockerfile (hunk 1 above)
Comment (mikem-zed) [reply tools/dockerfile-add-scanner/Dockerfile]: I fixed it, beside that I moved Dockerfile into /tools so we can add more tools down the road

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**